src/current/v23.2/show-ranges.md

```diff
@@ -77,6 +77,36 @@ Field | Description | Emitted for option(s)
 `index_end_key` | The end key of the last [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) of [index]({% link {{ page.version.version }}/indexes.md %}) data. | `INDEXES`
 `raw_index_start_key` | The start key of the first [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) of [index]({% link {{ page.version.version }}/indexes.md %}) data, expressed as [`BYTES`]({% link {{ page.version.version }}/bytes.md %}). | `INDEXES`, `KEYS`
 `raw_index_end_key` | The end key of the last [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) of [index]({% link {{ page.version.version }}/indexes.md %}) data, expressed as [`BYTES`]({% link {{ page.version.version }}/bytes.md %}). | `INDEXES`, `KEYS`
+`span_stats` | A `JSON` object containing span statistics. For more details, see [Span Statistics](#span-statistics). | `DETAILS`
+
+### Span Statistics
+
+CockroachDB stores all user data in a sorted map of key-value pairs, also known as a keyspace. A span refers to an interval within this keyspace.
+
+{% include_cached new-in.html version="v23.2" %} The `SHOW RANGES` command emits span statistics when the `DETAILS` option is specified. The statistics are included in a column named `span_stats`, as a `JSON` object.
+
+The statistics are calculated for the identifier of each row. For example,
+
+- `SHOW RANGES WITH DETAILS` will compute span statistics for each [range]({% link {{ page.version.version }}/ui-replication-dashboard.md %}#review-of-cockroachdb-terminology).
+- `SHOW RANGES WITH TABLES, DETAILS` will compute span statistics for each table, and so on.
```

Comment:
What does "and so on" mean?

Does this mean it will compute the span statistics for each table and each range?